doc/src/code-generation.md

```diff
@@ -117,7 +116,9 @@ be accessed to drive the code generation.
 Please see the [api documentation](codegenapi:slick.codegen.SourceCodeGenerator) for info
 on all of the methods that can be overridden for customization.
 
-Here is an example for customizing the generator:
+Here is an example for customizing the generator. Noteworthy, the line `override def hugeClassEnabled = false` 
```

Comment:
so does this mean your change is not default?

Reply:
My change is default (`hugeClassEnabled = true`). I called a class with more than 22 parameters _huge_ (no idea if there is a better name). 

And `override def hugeClassEnabled = false` brings back what was default before Slick 3.3. See also the [Scaladoc](https://github.com/mb73/slick/blob/generateCaseClassesForAllTables/slick-codegen/src/main/scala/slick/codegen/AbstractGenerator.scala#L92).

Maybe you didn’t see that I changed the customisation code example and hence the confusion?

Reply:
Or is the word “also” in “This was also the default behaviour before the advent of Slick 3.3.” misleading?

Reply:
The new wording in 59d906f is better.